src/components/index/IndexComponent.tsx

```diff
@@ -59,6 +62,7 @@ const Index = ({
   const { state } = useClient();
   const { descendingRequests } = useReader(state);
   const [filteredRequests, setFilteredRequests] = useState(initialFR);
+  console.log({ descendingRequests, filteredRequests, state });
```

Comment:
Log?

Reply:
thats wierd, not showing up in my code, maybe i havent pushed